frontend/src/pages/org/browser-profiles-detail.ts

```diff
@@ -123,75 +130,69 @@ export class BrowserProfilesDetail extends TailwindElement {
-            <dt class="text-sm text-0-600">
-              <span class="inline-block align-middle"
-                >${msg("Created at")}</span
-              >
-            </dt>
-            <dd>
-              ${this.profile
-                ? html`
-                    <sl-format-date
-                      lang=${getLocale()}
-                      date=${`${this.profile.created}Z` /** Z for UTC */}
-                      month="2-digit"
-                      day="2-digit"
-                      year="2-digit"
-                      hour="numeric"
-                      minute="numeric"
-                      time-zone-name="short"
-                    ></sl-format-date>
-                  `
-                : ""}
-            </dd>
-          </div>
-          <div class="col-span-3 md:col-span-1">
-            <dt class="text-sm text-0-600">
-              <span class="inline-block align-middle"
-                >${msg("Crawl Workflows")}</span
-              >
-              <sl-tooltip content=${msg("Crawl workflows using this profile")}>
-                <sl-icon
-                  class="inline-block align-middle"
-                  name="info-circle"
-                ></sl-icon>
-              </sl-tooltip>
-            </dt>
-            <dd>
-              <ul class="text-sm font-medium">
-                ${this.profile?.crawlconfigs?.map(
-                  ({ id, name }) => html`
-                    <li>
-                      <a
-                        class="text-neutral-600 hover:underline"
-                        href=${`${this.navigate.orgBasePath}/workflows/crawl/${id}`}
-                        @click=${this.navigate.link}
-                      >
-                        ${name}
-                      </a>
-                    </li>
-                  `,
-                )}
-              </ul>
-            </dd>
-          </div>
-        </dl>
+      <section class="mb-5 rounded-lg border px-4 py-2">
+        <btrix-desc-list horizontal>
+          <btrix-desc-list-item label=${msg("Backup Status")}>
+            <div class="flex items-center gap-2">
+              ${isBackedUp
+                ? html`<sl-icon
+                      name="clouds-fill"
+                      class="text-success"
+                    ></sl-icon>
+                    ${msg("Backed Up")}`
+                : html`<sl-icon
+                      name="cloud-slash-fill"
+                      class="text-neutral-500"
+                    ></sl-icon>
+                    ${msg("Not Backed Up")}`}
+            </div>
+          </btrix-desc-list-item>
+          <btrix-desc-list-item label=${msg("Crawler Release Channel")}>
+            ${this.profile
+              ? this.profile.crawlerChannel
+                ? capitalize(this.profile.crawlerChannel)
+                : none
+              : nothing}
+          </btrix-desc-list-item>
+          <btrix-desc-list-item label=${msg("Created At")}>
```

Comment:
Would be nice to add the `createdByName` and `modifiedByName` fields here if/when available. Might require rebasing this branch on latest main to test if using a fully local backend + frontend.

Reply:
What would a reasonable fallback text be if the names are unavailable, "unknown", or hide the field altogether?

Reply:
Maybe the latter? Only display the name if available

Reply:
cc @Shrinks99 ^

Reply:
Will handle in https://github.com/webrecorder/browsertrix/issues/478

Reply:
I like only display the name if available?